### PR TITLE
Add course instructor page for modules

### LINKED ISSUE #10658: Missing modules page
While working on #10653, it was realized that we do not currently have a page that that displays the modules for a course. We have one for assessment sets and should add something similar for modules.

### PR DESCRIPTION
**Pull request by mylesw27**
Closes #10658. While working on #10653, it was realized that we do not have a page that displays the modules for a course. This PR intends to add that page.
Screenshot 2024-09-13 at 2 19 09 PM

I have added columns from the `assessment_modules` table in the DB. If there is any other information that would add value here, please let me know.

### CODE REVIEW

File: apps/prairielearn/src/components/NavbarContext.html.ts [hunk 1]
```diff
@@ -140,6 +140,12 @@ const navPagesTabs: Partial<Record<Exclude<NavPage, undefined>, TabInfo[]>> = {
       iconClasses: 'fas fa-quote-right',
       tabLabel: 'Topics',
     },
+    {
+      activeSubPage: 'modules',
+      urlSuffix: '/course_admin/modules',
+      iconClasses: 'fa fa-layer-group',
```
Comment (mylesw27) [inline apps/prairielearn/src/components/NavbarContext.html.ts:146]: I used the `fa-layer-group` icon but was unsure if this is the best fit. Let me know if you have other suggestions for this icon. 
Screenshot 2024-09-13 at 2 22 35 PM

Comment (nwalters512) [reply apps/prairielearn/src/components/NavbarContext.html.ts:146]: I like this icon!

**Review by nwalters512 — APPROVED** [review]
This looks great! One small change to consider (I won't block on this): add a footer with a brief description and a link to the docs (https://prairielearn.readthedocs.io/en/latest/course/#assessment-modules).

**Comment by mylesw27** [comment]
> This looks great! One small change to consider (I won't block on this): add a footer with a brief description and a link to the docs (https://prairielearn.readthedocs.io/en/latest/course/#assessment-modules).

Added a footer with a brief description and a link to the docs in [b568bac](https://github.com/PrairieLearn/PrairieLearn/pull/10660/commits/b568bac74af039871321a66cc1a40dc830711e22).

**Review by jonatanschroeder — CHANGES_REQUESTED** [review]
The tabs should be in alphabetical order.

File: apps/prairielearn/src/pages/instructorCourseAdminModules/instructorCourseAdminModules.html.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                </tbody>
+              </table>
+            </div>
+            <div class="card-footer">
+              <small>
+                Modules can represent course topics, chapters, or any assessments of related
+                content. More information on modules can be found in the
+                <a href="https://prairielearn.readthedocs.io/en/latest/course/#assessment-modules">
+                  PrairieLearn documentation</a
+                >.
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/instructorCourseAdminModules/instructorCourseAdminModules.html.ts:61]: Nit (this is a little more specific about how they're used, not just what they are):

```suggestion
                Modules can be used to group assessments by topic, chapter, section, or other category. More information on modules can be found in the
                <a href="https://prairielearn.readthedocs.io/en/latest/course/#assessment-modules">
                  PrairieLearn documentation</a
                >.
```

**Review by nwalters512 — APPROVED** [review]

**Comment by mylesw27** [comment]
> The tabs should be in alphabetical order.

Moved tab to correct spot alphabetically in [aab802a](https://github.com/PrairieLearn/PrairieLearn/pull/10660/commits/aab802ab6ab00a62437cce5addc3aee54ea27f9c).

**Review by jonatanschroeder — APPROVED** [review]

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**